### PR TITLE
Updated cart functionality and styling

### PR DESCRIPTION
**Pull request by jessicaawhy**
Summary:

- Displayed cart data in a more consistent manner
- Updated cart controller to get cart items for a specific user
- Installed `@mantine/dates` for `DatePicker` component
- Added `DatePicker` to cart for each item to allow users to schedule delivery

### CODE REVIEW

File: client/context/UserContext.jsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const useUserUpdate = () => (
 );
 
 export const UserProvider = ({ children }) => {
-  const [User, setUser] = useState(null);
+  const [User, setUser] = useState(1);
```
Comment (david-deaton) [inline client/context/UserContext.jsx]: @jessicaawhy Is User 1 supposed to be Guest?

**Review by david-deaton — COMMENTED** [review]
Hi Jessica, I had a question re: the default User in the UserContext doc. Can you please review? Thanks!

File: client/context/UserContext.jsx (hunk 1 above)
Comment (jessicaawhy) [reply client/context/UserContext.jsx]: I was doing some testing and forgot to change this back. Good catch, thank you!

**Review by finchzebra — DISMISSED** [review]
Cart page looks great! Confirmed the following:

- The page renders aesthetically, with uniform layout, size, and alignment of details for each item in the cart
- Cart's API request passes a user parameter and retrieves entries from the database consistent with the designated user
- The customer can schedule a delivery date for each item, through use of Mantine/DatePicker

Please proceed.

**Review by david-deaton — APPROVED** [review]

**Pull request merged**